### PR TITLE
Makes Campaign Card Clickable

### PR DESCRIPTION
**Pull request by redreceipt**
## DESCRIPTION

Kapture 2019-05-15 at 11 54 44

### What does this PR do, or why is it needed?

The campaign card needs to be clickable. Currently, it's not.

### What design trade-offs/decisions were made?

Now it's just rendering a `ContentCard` and just passing props instead of a double network call.

### How do I test this PR?

Boot up the sim, click the first card.

### What automated tests did you write?

None.

## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [#694]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [x] Read through the "Files changed" tab _very carefully_
- [x] Edge cases: what assumptions are made about input?
- [x] What kind of tests could be written?
- [x] How might we make this easier for someone else to understand?
- [x] Could the code be simpler?
- [x] Will the code be easy to modify in the future?
- [x] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by redreceipt** [comment]
@vinnyjth can you give me a quick explanation of how I would figure out why this coverage is failing?

File: packages/apolloschurchapp/src/tabs/home/Home.js [hunk 1]
```diff
@@ -86,11 +88,33 @@ class Home extends PureComponent {
                           {}
                         );
 
+                        const metrics = [
```
Comment (conrad-vanl) [inline packages/apolloschurchapp/src/tabs/home/Home.js]: Not sure why `metrics` and `coverImage` is added to this component here......it shouldn't be. You're also changing the component from `ContentCardConnected` to `ContentCard`. Why?

**Comment by redreceipt** [comment]
Because using the connected component it was doing a double network call,
one with Isaac's query and one with the standard content query, I just cut
out the middle man. I agree with you and honestly I was hoping it would
spark a discussion. I see two options, combining the queries into one and
just using the content card connected, or making a separate component
called CampaignCard that would be styled a little different. Personally I
think this is the way to go long term because it needs to be different than
a standard content card. But obviously this would require a design. What do
you think?

--
Michael Neeley
Junior Web Developer
NewSpring Church
www.newspring.cc
Cell: (803) 622-6599

**Comment by conrad-vanl** [comment]
@redreceipt it shouldn't result in double network calls if the queries are set up right. Essentially, what you need to do is make sure you're querying for _all_ of the information that's inside of the `ContentCardConnected` query within the `Campaign` query, and then when the `ContentCardConnected` it will be able to read 100% from cache.

**Comment by redreceipt** [comment]
Not sure how we're gonna cover the diff because it's mainly just a `TouchableScale` with an `onPress` handler.

**Comment by redreceipt** [comment]
@conrad-vanl should I extend test coverage here? to make it pass? Not quite sure how if so.

**Comment by vinnyjth** [comment]
@redreceipt I wouldn't worry about it in this case, I think you've got as much covered as you can.

**Pull request merged**